### ISSUE TITLE
similar Prediction scores query

### ISSUE THREAD

**Issue by kshitij91** [tensorflow/ranking#154]
I have added multiple features to examples at the time of training:

1. When i predict scores for my input, giving the feature values resulting scores are good and works fine but when I dont give features as input to predict the scores predicted are similar.

I know this is because by default, it takes features as 0 or []. Is it necessary to give feature values at the time of prediction? Is there any way in which I can go ahead and not give features as input during prediction and get good scores ?

**Comment by kshitij91** [comment]
Also, if I remove features from training set and train the model using 'document_tokens' and 'relevance' label only, I am getting similar scores for the list at the time of prediction. Is this expected? What should I check in this case?

**Comment by ramakumar1729** [comment]
@kshitij91 : I'm not sure what you mean by not giving features as input. If you do not supply features, it will treat all documents the same, and give the same score. If you are getting similar scores, it could also mean your model isn't learning anything useful.

**Comment by ramakumar1729** [comment]
@kshitij91 : Does my previous reply address your issue? Let me know if you need any further clarification, or if this issue is resolved.

**Comment by ramakumar1729** [comment]
Closing this due to inactivity. Please reopen or file a new issue if needed.
